pkg/ddc/alluxio/ufs.go

```diff
@@ -90,6 +92,19 @@ func (e *AlluxioEngine) ShouldUpdateUFS() (ufsToUpdate *utils.UFSToUpdate) {
 	ufsToUpdate = utils.NewUFSToUpdate(dataset)
 	ufsToUpdate.AnalyzePathsDelta()
 
+	// 3. for hostpath ufs mount, check if all mountpoints have been mounted
+	const envVar = "FLUID_ENABLE_REMOUNT_DURING_SYNC"
```

Comment:
1. I think the const can be put at somewhere else with other consts. Also, `envVar` seems not a self-clarified const name.
2. How to set this environment? Do we need to provide an option for users to enable this feature?

Reply:
agree on 1. 
For 2, I think maybe we can set this Env to be true in the newest release and turn it off if it show bad effect.
And remove check logic after 2 releases.